### ISSUE TITLE
403 on some sites in v2.x

### ISSUE THREAD

**Issue by slobodangacesa** [itteco/iframely#548]
I'm running v1.6.1 and v2.x (tried multiple versions, including 2.4.x) in parallel. Both are with default configs.
on v2 I've noticed that I'm getting 403 for some sites, where they work just fine on v1.x.

both were build as docker images with default Dockerfile from project.

example:
`curl localhost:8061/iframely?url=https://www.bbc.com
{"error":{"source":"iframely","code":403,"message":"Forbidden"}}`

Also, stock config doesn't work with fix for EU shorts, so I just comment that out in config, but is there a way to make that work?

> iframely  | file:///iframely/config.local.js:260
> iframely  |             fix_shorts_in_eu: true              // Avoid consent redirect for EU servers
> iframely  |             ^^^^^^^^^^^^^^^^
> iframely  |
> iframely  | SyntaxError: Unexpected identifier
> iframely  |     at ESMLoader.moduleStrategy (node:internal/modules/esm/translators:117:18)
> iframely  |     at ESMLoader.moduleProvider (node:internal/modules/esm/loader:337:14)
> iframely  |     at async link (node:internal/modules/esm/module_job:70:21)
> iframely  |
> iframely  | Node.js v17.8.0
> iframely exited with code 0

**Comment by iparamonau** [comment]
The config issue is fixed by #558. 

The 403 issues depend on your server internet provider and on publishers that you try to reach. There is nothing in this package that affects those errors. You may try to change the default user-agent string though, see example config.

**Comment by cake-devops** [comment]
Thanks a lot for config fix!

I'm testing v1 vs v2 from same network, and I've tried multiple: AWS, Hetzner, GCP, few local ISPs.
I have also tried setting user agent from v1 into v2, and also setting completely different user agent (Chrome and Firefox).
Every time I get 403 for www.bbc.com in v2, but it works in v1.
Freshly built docker image from dev branch works pretty much the same like v2.

**Comment by slobodangacesa** [comment]
Thanks a lot for config fix!

I'm testing v1 vs v2 from same network, and I've tried multiple: AWS, Hetzner, GCP, few local ISPs.
I have also tried setting user agent from v1 into v2, and also setting completely different user agent (Chrome and Firefox).
Every time I get 403 for www.bbc.com in v2, but it works in v1.
Freshly built docker image from dev branch works pretty much the same like v2.
